### ISSUE TITLE
UI: create hi-res screen for exploration.

### ISSUE THREAD

**Issue by ZaneDubya** [ZaneDubya/CommunityKitPublic#884]
- [x] Mechanics pass begins!
   - [x] class UiViewDungeonExplore : UiViewDungeon : UiAScreen
   - [x] class UiDungeonPOV
   - [x] class TwUiDungeonPOV : UiDungeonPOV
   - [x] Mini Map control
   - [x] Inventory window
   - [x] Loot window - separate from inventory?
   - [x] Character stats and equipment window
   - [x] Spells window
   - [x] Skills window
   - [x] Options
   - [x] Add mouse over information to Skills/Spells (with IconInputWithToolTip class).
   - [x] Players/Party formation window (replaces map/players/party window)
   - [x] Character social window
     - [x] Achievements - invoked from social window.
     - [x] Hi-res achievements
   - [x] Mini map shows 'special' tiles.
   - [x] Name/Health/Mana Pop-up on mouse over party tiles.
   - [x] Portrait pop-up
   - [x] Figure out how I'm going to handle trash... how about with an awesome pop up?!
   - [x] Better window switching mechanism in HiRes mode.
     - [x] Invoke achievement window from Dungeon. Use this in the Character Social window.
   - [x] Mechanics pass complete!
 - [x] Polish Pass begins!
   - [x] Mini Map
   - [x] Party tiles.
   - [x] Inventory
   - [x] Loot Window
   - [x] Char stats/equip
   - [x] Spells/Skills
   - [x] Options
   - [x] Character social/achievements
   - [x] Players/Party
   - [x] Clean up party tile invocation.

**Comment by ZaneDubya** [comment]
Screen off to a great start. 75% complete.

**Comment by ZaneDubya** [comment]
Equipment in progress.

Already thinking about what to do with the polish pass. How closely should the hi-res ui resemble the original UI?

**Comment by ZaneDubya** [comment]
Equipment done, first pass.

**Comment by ZaneDubya** [comment]
Spells and skills done, first pass.

**Comment by ZaneDubya** [comment]
Polish pass completed on spells and skills. Looks phenomenal.

**Comment by ZaneDubya** [comment]
Working on players/party window. Can I finish this today?

**Comment by ZaneDubya** [comment]
Finished players/party window, character social window, quality of life improvements for party tiles, achievements window. Will finish tomorrow.

**Comment by ZaneDubya** [comment]
Starting on window invocation. Hard.

**Comment by ZaneDubya** [comment]
Window invocation ready to go. To be honest the code is a complete cluster. This was some of the first code that I wrote when I began working on MedievaLands in 2018. I had no direction, and I just kind of added code as I went. Now, on top of that, it is supporting two different resolutions with completely different control sets. When I think about it my heart sinks.

But I'm ready to move on. All the mechanics are done. Time to polish!
